php/php.editor/test/unit/data/testfiles/formatting/broken/issue197074_04.php.formatted

```diff
@@ -21,25 +21,28 @@ class companyAddressesActions extends myFrontModuleActions {
         }
 
         if ($this->getPage()->getModuleAction() == 'supplier/editCompanyAddress')
-            $this->getRequest ()->setParameter ('cancel_url', $this->getHelper ()main/checkout')
+        $this->getRequest ()->setParameter ('cancel_url', $this->getHelper ()main/checkout')
```

Comment:
The cause of this change is I added `ASTErrroExpression` to the parameter (grammar file).
Original issue is https://bz.apache.org/netbeans/show_bug.cgi?id=197074

formatting behavior is changed, but original code has syntax error and the purpose of the original issue is to prevent AssertionError. So, I suppose there is no problem.

Reply:
Agreed.